sky/templates/spot-controller.yaml.j2

```diff
@@ -2,8 +2,9 @@
 
 name: {{cluster_name}}
 
-resources:
-  disk_size: 50
+# The resources will be defined in sky.spot_launch
```

Comment:
Hmm, are L7-12 still parsed or in effect? If not, should we remove?

Reply:
Good point. Moved the comment to the spot_utils.